### PR TITLE
feat: added increment and setIfNotExists APIs

### PR DESCRIPTION
**Pull request by nidhidham**

### CODE REVIEW

File: src/Momento.Sdk/ISimpleCacheClient.cs [hunk 1]
```diff
@@ -195,6 +195,37 @@ public interface ISimpleCacheClient : IDisposable
...
+    ///
+    /// <para>Associates the given key with the given value. If a value for the key is
+    /// already present it is not replaced with the new value.</para>
+    /// </summary>
+    /// <param name="cacheName">Name of the cache to store the key and field in.</param>
+    /// <param name="key">The key to set.</param>
+    /// <param name="field">The value to be stored.</param>
+    /// <param name="ttl">TTL for the field in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.</param>
+    /// <returns>Task representing the result of the cache operation.</returns>
+    public Task<CacheSetIfNotExistsResponse> SetIfNotExistsAsync(string cacheName, string key, string field, TimeSpan? ttl = null);
```
Comment (cprice404) [inline src/Momento.Sdk/ISimpleCacheClient.cs:209]: I think `field` should be named `value` here, and you probably want a second signature of this that will accept `byte[]`?

**Review by cprice404 — APPROVED** [review]
LGTM!  I think there's a couple of minor fixes in the interface definition but otherwise gtg

**Review by cprice404 — COMMENTED** [review]
lgtm, just a couple of minor things in the interface!

File: src/Momento.Sdk/ISimpleCacheClient.cs (hunk 1 above)
Comment (nidhidham) [reply src/Momento.Sdk/ISimpleCacheClient.cs:209]: Yup, will fix, thanks!

Comment (nidhidham) [reply src/Momento.Sdk/ISimpleCacheClient.cs]: Fixed

**Review by malandis — APPROVED** [review]

**Review by cprice404 — APPROVED** [review]

**Pull request merged**